thingif/src/main/java/com/kii/thingif/command/Action.java

```diff
@@ -7,6 +7,7 @@
 /**
  * Marks a class as a single action. The action class should implement this interface and have only
  * one field as value of the action.
+ * **Note** SDK doesn't allow register an non static inner class as an action class.
```

Comment:
Like https://github.com/KiiPlatform/thing-if-AndroidSDK/pull/293/commits/7a9d9043111114f052dad0290a47c74292d76224#r109102170 , I prefer to list accepting class.